### ISSUE TITLE
Using a list of lines for prerotate breaks formatting

### ISSUE THREAD

**Issue by UlrichEckhardt** [sous-chefs/logrotate#88]
The generated output is:

```
  prerotate
  # do stuff
# More info about stuff here.
stuff.sh --fou --barre
  endscript
```

It should be this:

```
  prerotate
    # do stuff
    # More info about stuff here.
    stuff.sh --fou --barre
  endscript
```

The differences are:
- All lines of the script should be indented the same.
- All lines of the script should be indented one level more than the surrounding `prerotate` and `endscript` tags.

**Comment by stevendanna** [comment]
👍 This weekend I'm going to work on moving this cookbook over to using resources rather than definitions.  I'll be sure to keep this in mind.

**Comment by stevendanna** [comment]
I thought I had included a fix for this in 2.0 but it seems not.  Marking this for 2.2.
